src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -15,7 +15,7 @@
  *    limitations under the License.
  */
 
-#import <Matter/MTRDefines.h>
+#import <Matter/Matter.h>
```

Comment:
This is the wrong way to import things, I'll fix this in a follow-on PR with general xcconfig and PCH (precompiled headers). This is a quick general patch.